app/controllers/concerns/attachment_manageable.rb

```diff
@@ -2,7 +2,24 @@ module AttachmentManageable
   def allow_only_one_attachment
     return unless params[:files].is_a?(Array)
 
-    flash.now[:alert] = t(".attachment_manageable.upload_limit_exceeded")
+    flash.now[:alert] = t("attachment_manageable.upload_limit_exceeded")
     render turbo_stream: turbo_stream.replace("flash", partial: "layouts/shared/flash_messages")
   end
+
+  def show_alert_if_attachment_missing
+    if !params[:pet][:images].nil?
```

Comment:
I think it would be more idiomatic to use
`if params[:pet][:images].present?` rather than a double negative

Reply:
Did this in [refactor: use present? instead of nil?](https://github.com/rubyforgood/homeward-tails/pull/1273/commits/2454e2d0883c9115aa2f2132d03d65f9dd1ba5a8)